lib/models/autodecoders/base_nerf.py

```diff
@@ -414,6 +418,7 @@ def inverse_code(self, decoder, cond_imgs, cond_rays_o, cond_rays_d, dt_gamma=0,
         with module_requires_grad(decoder, False):
             n_inverse_steps = cfg.get('n_inverse_steps', 1000)
             n_inverse_rays = cfg.get('n_inverse_rays', 4096)
+            n_inverse_rays_multi = cfg.get('n_inverse_rays_multi', 4096)
```

Comment:
Let's name this so that it is clear that those rays are used for consistency loss